### PR TITLE
Service/maker

### PR DESCRIPTION
**Pull request by TanieciaDuplantis**

### CODE REVIEW

File: src/app/maker.service.ts [hunk 1]
```diff
@@ -1,26 +1,20 @@
 import { Injectable } from '@angular/core';
 import data from 'mockDataMaker.json';
+import { makeBindingParser } from '@angular/compiler';
```
Comment (alexa-w) [inline src/app/maker.service.ts:3]: What is makeBindingParser doing?

File: src/app/maker/maker-navbar/maker-navbar.component.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { Component, OnInit, Output, EventEmitter } from '@angular/core';
   templateUrl: './maker-navbar.component.html',
   styleUrls: ['./maker-navbar.component.scss'],
 })
-export class WerkerNavbarComponent implements OnInit {
+export class MakerNavBarComponent implements OnInit {
```
Comment (alexa-w) [inline src/app/maker/maker-navbar/maker-navbar.component.ts:8]: :+1:

File: src/app/maker/maker-profile/maker-profile.component.html [hunk 3]
```diff
@@ -4,15 +4,15 @@ <h3>Your Maker Profile</h3>
   <ion-item enter-maker>
     <ion-label>Name:</ion-label>
     <ion-input placeholder="Enter first name"></ion-input><ion-input placeholder="and last name"></ion-input>
-    <input placeholder="Enter first name"></input><input placeholder="and last name"></input>
```
Comment (alexa-w) [inline src/app/maker/maker-profile/maker-profile.component.html:7]: :+1:

File: src/app/maker/maker.module.ts [hunk 4]
```diff
@@ -21,6 +34,19 @@ const routes: Routes = [
     IonicModule,
     RouterModule.forChild(routes)
   ],
-  declarations: [MakerPage]
```
Comment (alexa-w) [inline src/app/maker/maker.module.ts:24]: nice!

File: src/app/maker.service.ts [hunk 5]
```diff
@@ -1,26 +1,20 @@
...
-   * gets maker from present data by id
-   * @todo make network request
-   * @todo return Promise<Maker>
-   *
-   * @param id - db-generated id
+   * gets Maker from database by id
    */
-  getMakerById(id: Number) {
-    return this.allMakers.find(maker => maker.id === id);
+  getMakerById(id: Number): Object {
```
Comment (alexa-w) [inline src/app/maker.service.ts:17]: I assume you don't mean for this to make a network request yet

**Comment by frankdamico** [comment]
ok

**Pull request closed without merging**